chromium_src/third_party/blink/renderer/core/execution_context/execution_context.cc

```diff
@@ -221,11 +212,6 @@ BraveSessionCache::PerturbPixelsInternal(
       v = lfsr_next(v);
     }
   }
-  // convert back to a StaticBitmapImage to return to the caller
-  scoped_refptr<blink::StaticBitmapImage> perturbed_bitmap =
-      blink::UnacceleratedStaticBitmapImage::Create(
-          data_buffer->RetainedImage());
```

Comment:
> RetainedImage is owned by ImageDataBuffer which goes out of scope at the end of the method. Also returning a new static bitmap image is potentially problematic for memory.

Are there really any problems? `RetainedImage()` is a shared_ptr and from `UnacceleratedStaticBitmapImage` it seems that only the pointer will be copied - see `PaintImage::CreateSkImage`

Reply:
it wasn't totally clear, but either way there's no reason to do all of this

Reply:
I guess we then should change the issue description, because this refactoring fixes something different than what is decribed